packages/kit/src/core/config/index.spec.js

```diff
@@ -38,6 +38,9 @@ test('fills in defaults', () => {
 					exclude: []
 				}
 			},
+			serviceWorker: {
+				filesExclusions: []
```

Comment:
I would probably just call this option `exclude`

Reply:
Done.